mwdb/web/src/components/Upload.js

```diff
@@ -10,6 +10,44 @@ import api from "@mwdb-web/commons/api";
 import { AuthContext } from "@mwdb-web/commons/auth";
 import { DataTable, View } from "@mwdb-web/commons/ui";
 
+function UploadDropzone(props) {
+    const onDrop = useCallback(acceptedFiles => props.onDrop(acceptedFiles[0]), [])
```

Comment:
missing dependency
```suggestion
    const onDrop = useCallback(acceptedFiles => props.onDrop(acceptedFiles[0]), [props])
```

Reply:
I think `props` is too broad. Destructured props.onDrop into separate var.